src/background.ts

```diff
@@ -560,6 +562,59 @@ chrome.commands.onCommand.addListener(async (command: string) => {
       });
       break;
 
+    case "autofill":
+      await new Promise(
+        (resolve: () => void, reject: (reason: Error) => void) => {
+          try {
+            return chrome.tabs.executeScript(
+              { file: "/dist/content.js" },
+              () => {
+                chrome.tabs.insertCSS({ file: "/css/content.css" }, resolve);
+              }
+            );
+          } catch (error) {
+            console.error(error);
+            return reject(error);
+          }
+        }
+      );
+
+      if (cachedPassphrase === "") {
```

Comment:
This can be removed since you account for unencrypted entries.

Reply:
If the extension is locked, and the user presses the hotkey to auto-fill code, this code block can prevent the command. Do I understand correctly?

Reply:
Yes, but doesn't this also prevent unencrypted access?

Reply:
Oh, I got your point. Let me update this.

Reply:
Updated